code/datums/traits/negative.dm

```diff
@@ -38,9 +38,17 @@
 	var/obj/item/heirloom_type
 	switch(quirk_holder.mind.assigned_role)
 		if("Clown")
-			heirloom_type = /obj/item/bikehorn/golden
+			heirloom_type = /obj/item/paint/anycolor
 		if("Mime")
-			heirloom_type = /obj/item/reagent_containers/food/snacks/baguette
+			heirloom_type = /obj/item/toy/dummy
+		if("Cook")
+			heirloom_type = /obj/item/kitchen/knife
```

Comment:
Or something more interesting than a knife

Reply:
```suggestion
			heirloom_type = /obj/item/kitchen/rollingpin
```
why not

Reply:
```suggestion
			heirloom_type = /obj/item/clothing/head/chefhat
```